reactor-core/src/main/java/reactor/core/publisher/Traces.java

```diff
@@ -129,48 +128,88 @@ static boolean isUserCode(String line) {
 	 * from the assembly stack trace.
 	 */
 	static String[] extractOperatorAssemblyInformationParts(String source) {
```

Comment:
Since you're changing this mechanism a lot, can you also review the javadoc and update? E.g. `skipFirst` remained in the javadoc while there is no such argument here.